security/sec-iride/src/test/java/org/geoserver/security/iride/identity/IrideIdentityTokenizerTest.java

```diff
@@ -80,7 +79,7 @@ public void testTokenizationThrowNullPointerExceptionForNullValue() throws Iride
      */
     @Test(expected = IrideIdentityMissingTokenException.class)
     public void testTokenizationFailForEmptyOrBlankValue() throws IrideIdentityMissingTokenException {
-        final String value = StringUtils.repeat(" ", RandomUtils.nextInt(10));
+        final String value = randomBlankOrEmptyString();
```

Comment:
@mbarto the tests using those random (from 0 to 10) length strings are, in my opinion, still predictable: the only randomic part being the input_length_, which can go between 0 (inclusive) and 10 (non-inclusive).

That said I can split those tests relying on random length strings in three simpler tests:
- one for `null` string input
- one for "empty" string input
- one for "blank" string input